utils/config/daos_server.yml

```diff
@@ -304,7 +304,7 @@
 #
 #  targets: 20
 #
-#  # Number of helper XStreams per VOS server.
+#  # Number of helper XStreams per VOS engine.
```

Comment:
Not sure if this particular substitution is correct. @jolivier23: Does this new terminology make sense? That having been said, I'm not sure that "VOS server" made sense either.

Reply:
Right, there is no such thing as VOS server. The original text did not make sense.
That's the number of I/O service threads (and network endpoints) per engine.

Reply:
can fix this text in https://github.com/daos-stack/daos/pull/4611